app/src/components/market/sections/market_bond/market_bond.tsx

```diff
@@ -44,6 +44,7 @@ interface Props extends RouteComponentProps<any> {
 const BottomButtonWrapper = styled(ButtonContainer)`
   margin: 0 -24px;
   padding: 20px 24px 0;
+  ${props => (props.marginTop ? '20px' : '0')}
```

Comment:
@Frankaus So here you don't need this props since inside the ButtonContainer there is already marginTop which should be reused...

Reply:
right! Thanks mate